packages/openapi-fetch/test/v7-beta.test.ts

```diff
@@ -0,0 +1,885 @@
+import { atom, computed } from "nanostores";
+import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
+// @ts-expect-error
+import createFetchMock from "vitest-fetch-mock";
+import createClient from "../src/index.js";
+import type { paths } from "./v7-beta.js";
+
+// Note
+// This is a copy of index.test.ts, but uses generated types from openapi-typescript@7 (beta).
+// This tests upcoming compatibility until openapi-typescript@7 is stable and the two tests
+// merged together.
```

Comment:
Temporary 2nd test file added to make sure openapi-typescript@7 support is easy when the time comes.